doc/connectivity/bluetooth/audio.rst

```diff
@@ -0,0 +1,384 @@
+#########
+BLE Audio
+#########
+
+Presentation Layer
+==================
+
+To distribute and/or consume audio using BLE a presentation layer is
+required between the user application and the ISO data path. The main
+roles of the presentation layer are:
+
+* To encode/decode raw audio data using the LC3 codec.
+* To synchronize the audio playback/recording with the BLE clock.
+* To synchronize the audio playback/recording among all the audio sinks/sources.
+
+Typical LE audio applications
+=============================
+
+To better understand the responsibilities of the presentation layer,
+this section describes some examples of typical LE audio applications,
+from the simplest to the most intricate.
+
+Unidirectional, single source, single sink
+------------------------------------------
+
+BLE can be used to transmit stereo audio from a source device to a
+headset. The source device and headset are connected using a single
```

Comment:
Is it just headset, or more wider range like "an audio output device, e.g. headset." or something like that? If changed, then please check also other "headset" instances in this same chapter.

Reply:
_Headset_ is shorter so I'll stick to it :) This is a use case description basically.